File: Programación y Algoritmos/Actividad 13/auto.cpp
#include <iostream>
#include <string>

using namespace std;

float tarifa;
float kilometros;

int main() 
{
    tarifa = 850;
    cout << "Introduzca la cantidad de kilometros recorridos" << endl;
    cin >> kilometros;
    
    if (kilometros > 200)
    {
        tarifa = tarifa + (4.50 * (kilometros - 200));
    }

    cout << "Monto a pagar: $" << tarifa << endl;
    cout << "IVA: $" << tarifa*.16 << endl;
    cout << "Monto total: $" << tarifa*1.16 << endl;
}

File: Programación y Algoritmos/Actividad 13/auto.py
print("Introduzca la cantidad de kilómetros recorridos")

try: 
    kilometros = float(input())
except (ValueError):
    print("Ese no es un número")

tarifa = 850

if kilometros > 200:
    tarifa = tarifa + ((kilometros - 200 )* 4.50)

print(f"Monto a pagar: ${tarifa:.2f}")
print(f"IVA: ${tarifa*.16:.2f}")
print(f"Monto total: ${tarifa*1.16:.2f}")